tests/test_hotjar.py

```
@@ -37,7 +35,4 @@ def test_that_plugin_loads_hotjar():
     response = app_with_plugin.test_client().get("/")
     assert response.status_code == 404
     decoded_response = response.data.decode()
-    assert (
-        hotjar_function in decoded_response
-        and secret_id_for_testing in decoded_response
-    )
+    assert hotjar_function in decoded_response and secret_id_for_testing in decoded_response
```

Ruff (0.15.0)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)